tensorflow/python/ops/ctc_ops.py

```diff
@@ -180,8 +180,8 @@ def ctc_greedy_decoder(inputs, sequence_length, merge_repeated=True):
   only the first of these is emitted.  The sequence `A B B * B * B` (where '*'
   is the blank label) becomes
 
-    * `A B` if `merge_repeated=True`.
-    * `A B B B B B` if `merge_repeated=False`.
+    * `A B B B` if `merge_repeated=True`.
```

Comment:
In fact it is really confusing since from docs it seems that there is no way to obtain two similar labels in a row like `B B A` (that is one of the reasons why we need `blank`) which is not true for CTC.